src/components/SetBatchPoster.tsx

```diff
@@ -48,7 +49,7 @@ export function SetBatchPoster({ onNext }: { onNext: () => void }) {
   async function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
     event.preventDefault();
 
-    if (chain?.unsupported) {
+    if (chain?.id !== ChainId.ArbitrumGoerli) {
```

Comment:
NIT: It's being reused 3 times, I would extract it to `isOnSupportedChain`, if we ever need to change the condition, it's also easier to update